tutorials/k8s_workload.md

```diff
@@ -0,0 +1,352 @@
+
+
+### 6. Define Deployment Specification  
+
+### 6.1. Register the Deployment Pod Port  
+Because we are using the NSC SDN Controller stub plugin we must register the port element for the deployed VNF pod. This can be done using the test API:  
+* `POST https://10.3.205.173:8090/nsc/controller/10.3.205.177/networkElements/1` with json payload:  
+```json
+{
+"elementId":1,
+"parentId":"1",
+"deviceOwnerId":"default:testds",
+"macAddresses":  ["00:00:00:00:00:01"],
+"portIPs": ["70.70.0.1"]
+}
+```
+
+> Note:  This step is not needed if you are using a real SDN controller plugin (as opposed to a stub) like the *OSC Nuage plugin*. The SDN controller should already have and be able to provide the pod port to OSC after a pod is created on Kubernetes.  
+
+### 6.2 Create the Deployment Specification  
+
+Using the OSC api-doc `https://OSC_IP:8090/api-doc -> Operations for Virtual Systems`: 
+* `POST /api/server/v1/virtualSystems/{vsId}/deploymentSpecs` with json payload:  
+**vsId**: `VIRTUAL_SYSTEM_ID`  
+```json
+{
+ "name": "testds", 
+ "namespace": "default", 
+ "count": 1
+}
+```
+
+On the OSC UI you should see a deployment spec `testds` with the **Last job Status** **PASSED**.  
+**Troubleshooting:**   
+If the job failed perform the following steps to troubleshoot:
+1. Ensure that you are able to create pods on the Kubernetes cluster using the image `corfr/tcpdump`.  
+2. Ensure that OSC can communicate with the Kubernetes API server through the [proxy](/gettingstarted/requirements.md#user-content-kubernetes).  
+
+
+### 7. Define Security Group  
+
+#### 7.1. Register the Protected Pod Port  
+Because we are using the NSC SDN Controller stub plugin we must register the port element for the protected pod. This can be done using the test API:  
+* `POST https://10.3.205.173:8090/nsc/controller/10.3.205.177/networkElements/2` with json payload:  
+```json
+{
+"elementId":2,
+"parentId":"1",
+"deviceOwnerId":"default:victim-busybox",
+"macAddresses":  ["00:00:00:00:00:02"],
+"portIPs": ["70.70.0.2"]
+}
+```
+
+> Note:  This step is not needed if you are using a real SDN controller plugin (as opposed to a stub) like the *OSC Nuage plugin*. The SDN controller should already have and be able to provide the pod port to OSC after a pod is created on Kubernetes.  
+
+#### 7.2. Create the Security Group
+Using the OSC api-doc `https://OSC_IP:8090/api-doc -> Operations for Virtualization Connectors`:  
+* `POST /api/server/v1/virtualizationConnectors/{vcId}/securityGroups` with json payload:  
+**vcId**: `VIRTUALIZATION_CONNECTOR_ID`  
+```json
+{
+"name": "testSG"
+}
+```
+You should get a response body containing the ID of the create security group, similar to  `"id": SECURITY_GROUP_ID`, copy this identifier to be used on later steps.
+
+#### 7.3. Add a Label Member  
+Using the OSC api-doc `https://OSC_IP:8090/api-doc -> Operations for Virtualization Connectors`:  
```

Comment:
hightlight the fact that the label specified here matches the label of the victim/workload